index.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+    <html lang="en">
+
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
+        <link rel="stylesheet" type="text/css" href="main.css">
+        <title>Sing in project</title>
+    </head>
+
+    <body>
+        <div id="container">
+            <div id="col-1"></div>
+            <div id="col-2">
+                <div id="content-top">
+                    <div>
+                        <img src="./img/intuit_logo.jpg" alt="intuit" id="logo">
+                    </div>
+                    <div id="items">
+                        <ul>
+                            <li><a href="#"> <img src="./img/mint.jpg" alt=""></a> </li>
+                            <li><a href="#"><img src="./img/quickboks.jpg" alt=""></a></li>
+                            <li><a href="#"><img src="./img/turbotax.jpg" alt=""></a></li>
+                        </ul>
+                    </div>
+                </div>
+                <div id="form-container">
+                    <div>
+                        <div>
+                            <h1>Create an Intuit account</h1>
+                        </div>
+                        <div>
+                            <p id="txt-1">One account for averything Intuit, Including Ming.
+                        </div>
+                        <div>
+                            <a id="a-1" href="#">Learn more</a></p>
+                        </div>
+
+
+                    </div>
+
+                    <div>
+                        <form idaction="">
```

Comment:
`AWESOME` 

Nice job using `form` element, this makes the section represent a document section that contains interactive controls for submitting information to a web server. :+1: 

-----

`REQUIRED`

I would suggest using `action="/"` instead of providing empty value for the action attribute or deleting it completely as there's no request been made with the form. **Note that `idaction=""` is an invalid `form` attribute in this case.**